Content/.template.config/template.json

```diff
@@ -174,397 +170,345 @@
     },
     "sources": [
         {
-            "exclude": [ "**/.template.config/**/*", "**/paket*.lock" ],
+            "exclude": [
+                "**/.template.config/**/*",
+                "**/paket*.lock"
+            ],
             "modifiers": [
                 {
                     "include": "paket_noazure_bridge-nofulma-noreaction_saturn-bridge-noazure.lock",
                     "condition": "(server == \"saturn\" && remoting == false && bridge == true && deploy != \"azure\" && layout == \"none\" && reaction == false)",
-                    "rename": { "paket_noazure_bridge-nofulma-noreaction_saturn-bridge-noazure.lock": "paket.lock" }
+                    "rename": {

```

Comment:
these are auto generated from build script, and are not formatted - i.e. the generation script wouldn't use newlines here. Can we correct that so that future diffs are not misleading?

Reply:
Yes, sorry - that was just VS Code autoformatting.